src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represent a Person's contact in the address book.
```

Comment:
good! but not necessary to say address book; you can use this class in any application that deals with contact information :-)